src/devices/configured-device-card.ts

```diff
@@ -136,7 +136,7 @@ class ESPHomeConfiguredDeviceCard extends LitElement {
             <mwc-list-item>Download YAML</mwc-list-item>
             <mwc-list-item>Rename</mwc-list-item>
             <mwc-list-item>Clean Build Files</mwc-list-item>
-            <mwc-list-item>Delete</mwc-list-item>
+            <mwc-list-item class="destructive">Delete</mwc-list-item>
```

Comment:
For a future PR, if you're looking for things to contribute, let's add icons 👍

Reply:
done